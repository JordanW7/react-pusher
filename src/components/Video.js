import React from "react";
import "./Video.css";

const Video = () => {
  return (
    <section className="video">
      <iframe
        title="video"
        className="video-iframe"
        src="https://www.youtube.com/embed/XOacA3RYrXk"
        frameBorder="0"
        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
        allowFullScreen
      />
    </section>
  );
};

export default Video;
