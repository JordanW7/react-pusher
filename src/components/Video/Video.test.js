import React from "react";
import { shallow } from "enzyme";
import Video from "./Video";

it("expect to render Video component", () => {
  expect(shallow(<Video />)).toMatchSnapshot();
});
