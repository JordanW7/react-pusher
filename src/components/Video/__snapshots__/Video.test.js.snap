// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Video component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Video />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <iframe
        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
        allowFullScreen={true}
        className="video-iframe"
        frameBorder="0"
        src="https://www.youtube.com/embed/XOacA3RYrXk"
        title="video"
      />,
      "className": "video",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "allow": "accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture",
        "allowFullScreen": true,
        "className": "video-iframe",
        "frameBorder": "0",
        "src": "https://www.youtube.com/embed/XOacA3RYrXk",
        "title": "video",
      },
      "ref": null,
      "rendered": null,
      "type": "iframe",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <iframe
          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
          allowFullScreen={true}
          className="video-iframe"
          frameBorder="0"
          src="https://www.youtube.com/embed/XOacA3RYrXk"
          title="video"
        />,
        "className": "video",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "allow": "accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture",
          "allowFullScreen": true,
          "className": "video-iframe",
          "frameBorder": "0",
          "src": "https://www.youtube.com/embed/XOacA3RYrXk",
          "title": "video",
        },
        "ref": null,
        "rendered": null,
        "type": "iframe",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
